Makefile

```diff
@@ -9,9 +9,12 @@ BENCHTIMEOUT ?= 20m
 BENCHCOUNT ?= 1
 
 ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
+# Disable selinux for local podman builds.
+DOCKER_OPTS=--security-opt label=disable
+else
 # Podman DTRT by running processes unprivileged in containers,
 # but it's UID mapping is more nuanced. Only set user for vanilla docker.
-DOCKER_USER=--user "$(shell id -u)"
+DOCKER_OPTS=--user "$(shell id -u)"
```

Comment:
`DOCKER_USER` is still used:
https://github.com/stackrox/stackrox/blob/f613712fa03a6c8d6245658eaa9f284fd789dc4a/make/protogen.mk#L261

Reply:
Good catch, fixed